### PR TITLE
Add drawing task point tool model

### PR DESCRIPTION
**Pull request by mcbouslog**
Package: lib-classifier

Describe your changes:
One of a few tiny PRs related to adding the drawing task point tool. No required order to review/merge.
This adds the point tool model for the drawing store.

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/store/tasks/DrawingTask.js [hunk 1]
```diff
@@ -10,7 +11,14 @@ const Drawing = types.model('Drawing', {
   })),
   help: types.optional(types.string, ''),
   instruction: types.maybe(types.string),
-  tools: types.array(types.frozen()),
+  tools: types.array(types.union({
+    dispatcher: (snapshot) => {
+      switch (snapshot.type) {
+        case 'point':
+          return Point
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/tasks/DrawingTask.js]: Small stylistic point but you can surround cases in {} to isolate scope.
```js
case 'point': {
  return Point
}

File: packages/lib-classifier/src/store/tasks/drawingTools/Point.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { types, onSnapshot } from 'mobx-state-tree'
+
+const Point = types.model('Point', {
+  color: types.optional(types.string, '#00ff00'),
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/tasks/drawingTools/Point.js]: Do we want to code colours into the tools, or look them up from a shared lookup table?

Comment (mcbouslog) [reply packages/lib-classifier/src/store/tasks/drawingTools/Point.js]: Shared lookup table sounds better, but maybe I'm not thinking about this quite right. And it's subject to change if we change how the tools are built in the lab. Hm, maybe for now I just change to `types.maybe(types.string)` and can revisit.

Comment (srallen) [reply packages/lib-classifier/src/store/tasks/drawingTools/Point.js]: I think `types.optional(types.string, '')` might be better in this case? It depends on if `color` can ever be null or undefined.

File: packages/lib-classifier/src/store/tasks/DrawingTask.js [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
 const Drawing = types.model('Drawing', {
-  answers: types.array(types.frozen({
-    label: types.string,
-    next: types.maybe(types.string)
-  })),
   help: types.optional(types.string, ''),
   instruction: types.maybe(types.string),
-  tools: types.array(types.frozen()),
+  tools: types.array(types.union({
+    dispatcher: (snapshot) => {
```
Comment (simensta) [inline packages/lib-classifier/src/store/tasks/DrawingTask.js]: yarn lint detects an error on line 11: `11:28  error  Expected to return a value at the end of arrow function 'dispatcher'`.

Adding a `default` clause that returns `null` should resolve the error.

**Pull request merged**